installer/win/Brackets.wxs

```diff
@@ -77,6 +69,22 @@
                            Value="[INSTALLDIR]$(var.ExeName).exe,0" Type="string" />
             <RegistryValue Root="HKLM" Key="SOFTWARE\Classes\!(loc.ProductName) FileExt\shell\open"
                            Name="FriendlyAppName" Value="!(loc.ProductName)" Type="string" />
+
+            <!-- Add an entry to make sure it is runnable via Run Command  -->
+            <RegistryValue Root="HKLM" Key="SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\Brackets.exe" Value="[INSTALLDIR]$(var.ExeName).exe"
+                               Type="string" />
```

Comment:
The info at https://msdn.microsoft.com/en-us/library/windows/desktop/ee872121%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396 seems to suggest this is a pretty safe registry addition.